### ISSUE TITLE
Double axis values

### ISSUE THREAD

**Issue by worstenbrood** [nicolaskruchten/pivottable#777]
I get double count values on the y axis using this data

[{"Id":10632,"Parent":0,"Title":"qsdf","Contractor":"Abati","Score":"3","Observation date":"2017-07-04T22:00:00Z","Observation type":"Normal","Num Score":3.000000},{"Id":10631,"Parent":0,"Title":"qsdfdsf","Contractor":"GPA","Score":"1","Observation date":"2017-07-31T22:00:00Z","Observation type":"Normal","Num Score":1.000000},{"Id":10630,"Parent":0,"Title":"qsdf","Contractor":"Abati","Score":"3","Observation date":"2017-08-07T22:00:00Z","Observation type":"Normal","Num Score":3.000000},{"Id":10629,"Parent":0,"Title":"qsdfsdqf","Contractor":"Aaqua","Score":"5","Observation date":"2017-08-01T22:00:00Z","Observation type":"Normal","Num Score":5.000000},{"Id":10628,"Parent":0,"Title":"qsdf","Contractor":"GPA","Score":"4","Observation date":"2017-08-14T22:00:00Z","Observation type":"Normal","Num Score":4.000000},{"Id":10627,"Parent":0,"Title":"sqfqsd","Contractor":"GPA","Score":"3","Observation date":"2017-09-04T22:00:00Z","Observation type":"Normal","Num Score":3.000000},{"Id":10534,"Parent":0,"Title":"qsdf","Contractor":"Aaqua","Score":"2","Observation date":"2017-09-06T22:00:00Z","Observation type":"Normal","Num Score":2.000000},{"Id":10533,"Parent":0,"Title":"qfd","Contractor":"GPA","Score":"3","Observation date":"2016-09-06T22:00:00Z","Observation type":"Normal","Num Score":3.000000},{"Id":10531,"Parent":0,"Title":"qsdfd","Contractor":"GPA","Score":"3","Observation date":"2016-09-20T22:00:00Z","Observation type":"Normal","Num Score":3.000000},{"Id":10530,"Parent":0,"Title":"baqfsqdf","Contractor":"Abati","Score":"2","Observation date":"2015-07-14T22:00:00Z","Observation type":"Normal","Num Score":2.000000},{"Id":10529,"Parent":0,"Title":"test","Contractor":"Aaqua","Score":"4","Observation date":"2016-08-22T22:00:00Z","Observation type":"Technical","Num Score":4.000000},{"Id":10528,"Parent":0,"Title":"test","Contractor":"GPA","Score":"5","Observation date":"2015-09-22T22:00:00Z","Observation type":"Technical","Num Score":5.000000}]

![2017-09-11 13_45_35-list_ observation scores - tenforce](https://user-images.githubusercontent.com/304112/30273072-abb7017e-96f7-11e7-806a-22f8bf3ade81.png)

**Comment by nicolaskruchten** [comment]
Aye, that is ugly/unfortunate. C3js is trying to put in half-tickmarks (i.e 1.5, 2, 2.5) but the integer-only number-formatter that the Count aggregator passes in doesn't show the half-tickmarks. There might be some option we can pass in to C3 for this case. I'll look into it. Thanks for pointing this out!

**Comment by worstenbrood** [comment]
Correct, changed the usFmtInt to numberFormat() and values are showing correct now. This only seems to happen if you don't have enough values.
